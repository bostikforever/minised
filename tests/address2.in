line 1 PRINT
line 2
line 3 PRINT
and so on
