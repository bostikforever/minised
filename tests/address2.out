line 1 PRINT
line 3 PRINT
